### PR TITLE
Hello project

### PR DESCRIPTION
**Pull request by KayLemba**
In this project I did the following:
1. Add index.html
2. Add styles.css file
3. Modify styles.css and Add test.md file
4. Add test.md file to .gitignore
5. Modify README.md file
6. Add hello2.png file

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+this text should be ignored by git
```
Comment (carlos-ssh) [inline test.md]: - This file should be ignored by git

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @KayLemba , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 1 above)
Comment (KayLemba) [reply test.md]: Hello I have tried putting the test.md file in the .gitignore file but it seems like the test.md file is'nt being ignored I dont understand. What do I do?

Comment (KayLemba) [reply test.md]: Please explain do i need to change the text?

Comment (dasileker) [reply test.md]: > Please explain do i need to change the text?

No you need just to add this file to gitignore

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @KayLemba** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

- As previous TSE explained you need to add the **`test.md`** file to **`gitignore`** file as the requirement says
nnn


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 1 above)
Comment (KayLemba) [reply test.md]: I have done that already.

**Review by dasileker — APPROVED** [review]
### **Hi @KayLemba** 

### **Project Approved✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**